src/Raven.Studio/typescript/models/auth/licenseModel.ts

```diff
@@ -41,21 +41,28 @@ class licenseModel {
         }
 
         const dateFormat = "YYYY MMMM Do";
-        const expiration = moment(licenseStatus.SubscriptionExpiration);
-        const now = moment();
-        const nextMonth = moment().add(1, 'month');
+        const expiration = moment.utc(licenseStatus.SubscriptionExpiration);
+        const now = moment.utc();
+        const nextMonth = moment.utc().add(1, 'month');
         if (now.isBefore(expiration)) {
             const relativeDurationClass = nextMonth.isBefore(expiration) ? "" : "text-warning";
 
             const fromDuration = generalUtils.formatDurationByDate(expiration, true);
-            return `${expiration.format(dateFormat)} <br /><small class="${relativeDurationClass}">(${fromDuration})</small>`;
+            return `${expiration.format(dateFormat)} UTC ${this.getLicenseInfoIcon({ date: expiration, isExpired: false })}<br /><small class="${relativeDurationClass}">(${fromDuration})</small>`;
         }
 
         const expiredClass = licenseStatus.Expired ? "text-danger" : "";
         const duration = generalUtils.formatDurationByDate(expiration, true);
-        return `${expiration.format(dateFormat)} <br /><Small class="${expiredClass}">(${duration})</Small>`;
+
+        return `${expiration.format(dateFormat)} UTC ${this.getLicenseInfoIcon({ date: expiration, isExpired: true })}<br /><Small class="${expiredClass}">(${duration})</Small>`;
```

Comment:
Do we show the date in `UTC`?
Can we show in the local time of the browser? This is what we do in other places in the studio where we show dates.

Reply:
@ppekrol and I decided that here the date will be UTC with tooltip on info icon showing local and utc.